Duplicate permission step added to permission test

Valentina1133 opened the pull request:
### What does the PR do

Added steps to verify duplication of permission

CI run:
https://ci.status.im/job/status-desktop/job/e2e/job/manual/2170/

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | c9d9ab4c | [#1](https://ci.status.im/job/status-desktop/job/prs/job/linux/job/x86_64/job/tests-nim/job/PR-15292/1/) | 2024-06-21 05:15:10 | ~6 min | `tests/nim` | [:page_facing_up:`log`](https://ci.status.im/job/status-desktop/job/prs/job/linux/job/x86_64/job/tests-nim/job/PR-15292/1//consoleText) |

anastasiyaig commented on test/e2e/gui/objects_map/communities_names.py:
```diff
@@ -178,10 +178,12 @@
 update_StatusButton = {"checkable": False, "container": statusDesktop_mainWindow_overlay, "type": "StatusButton", "unnamed": 1, "visible": True}
 isAllowedToEditPermissionView_StatusListItemTag = {"container": mainWindow_editPermissionView_EditPermissionView, "type": "StatusListItemTag", "unnamed": 1, "visible": True}
 memberRoleLimitWarning = {"container": mainWindow_editPermissionView_EditPermissionView, "id": "duplicationPanel", "type": "WarningPanel", "unnamed": 1, "visible": True}
+editPermissionView_duplicationPanel_WarningPanel = {"container": mainWindow_editPermissionView_EditPermissionView, "id": "duplicationPanel", "type": "WarningPanel", "unnamed": 1, "visible": True}
```
maybe adding object name here?

Valentina1133 replied:
Added

anastasiyaig commented:
permission test is failing here

Valentina1133 commented:
> permission test is failing here

Yes, I see that test for 6th member role permission test failed (not the one I changed). Going to check what is wrong there

Valentina1133 commented:
> permission test is failing here

Ok. Fixed. There were 2 methods and 2 locators (1 - for limit warning, 1 for duplicate warning), but it's actually the same object. So I reworked it a bit so we use one for both warnings
